lib/mwUtil.js

```diff
@@ -182,6 +182,33 @@ mwUtil.isNoCacheRequest = (req) => req.headers && /no-cache/i.test(req.headers['
  */
 mwUtil.isNoStoreRequest = (req) => req.headers && /no-store/i.test(req.headers['cache-control']);
 
+/**
+ * Checks whether the request is an 'if-unmodified-since' request
+ * @param   {Object}    req
+ * @return  {boolean}
+ */
+mwUtil.isUnmodifiedSinceRequest = (req) => Boolean(req.headers && req.headers['if-unmodified-since']);
+
+/**
+ * Checks whether the response has been modified since the timestamp
+ * in `if-unmodified-since` header of the request
+ * @param  {Object} req the request
+ * @param  {Object} res the response
+ * @return {boolean}    true if content has beed modified
+ */
+mwUtil.isModifiedSince = (req, res) => {
```

Comment:
It's a bit surprising that a method called isModifiedSince() will ignored the if-modified-since header, and instead look at the is-unmodified-since header.

I suggest to either turn the logic around and make this isUnmodifiedSince, or make it check both conditions and rename to something like meetsTimestampConditions() or some such.

Reply:
I renamed the function just to avoid implementing more functionality.